op_crates/crypto/Cargo.toml

```diff
@@ -15,5 +15,11 @@ path = "lib.rs"
 
 [dependencies]
 deno_core = { version = "0.80.1", path = "../../core" }
-rand = "0.8.3"
+rand = "0.7.0"
+ring = { version = "0.16.20", features = ["std"] }
+rsa = { git = "https://github.com/RustCrypto/RSA", rev = "616b08d" }
```

Comment:
Note: Using via git because`RSAPrivateKey::new_with_exp` is not published to crates.io yet.

Reply:
Edit: Using my fork of the RSA crate to fix rand issues. 
Self ToDo: Change to RSA when fork is merged & released.

Reply:
Fixed